spec/rake_helper.rb

```diff
@@ -0,0 +1,43 @@
+# For testing rake tasks
+# From: https://github.com/eliotsykes/rails-testing-toolbox/blob/master/tasks.rb
```

Comment:
I'm asking the author about License and submit a PR to make it MIT license. Otherwise I think this is enough attribution? It's actually a great gist of code!